dracut.sh

```diff
@@ -2154,6 +2154,8 @@ if [[ $early_microcode == yes ]]; then
                 done
                 for i in $_fwdir/$_fw/$_src; do
                     [[ -e $i ]] || continue
+                    # skip README{.xz,.zst,...}
+                    str_starts "$i" "$_fwdir/$_fw/README" && continue
```

Comment:
this just puts a bandage on a symptom of the problem, instead of fixing the actual problem: the proper way would be to only `cat` files that match the glob `*.bin` (for amd) or `??-??-??` (for intel)